stylelint.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
 		'csstools/use-logical': [
 			'always',
 			{
-				severity: 'warning',
+				severity: 'error',
```

Comment:
We got more to fix on stable8. Set back to warning for now?

Reply:
```suggestion
				severity: 'warning',
```